### ISSUE TITLE
Learn More text cut off on Dev Drive card

### ISSUE THREAD

**Issue by cinnamon-msft** [microsoft/devhome#2093]
### Dev Home version

0.801.344.0

### Steps to reproduce the bug

Navigate to the Introducing Dev Home page

### Expected result

You should be able to see all the text in the card

### Actual result

![image](https://github.com/microsoft/devhome/assets/48369326/a2e03147-14d2-4922-9dbb-233e7283d1a1)

**Comment by krschau** [comment]
@cinnamon-msft what's your window size/shape? Is text scaling at 100%? High DPI? I'm unable to repro this.

**Comment by cinnamon-msft** [comment]
Res: 2496 x 1664
Scale: 150%
